### ISSUE TITLE
An unknown song plays while doing a backup

### ISSUE THREAD

**Issue by bondlxv** [federicoiosue/Omni-Notes#850]
I just upgraded OmniNotes to version 6.1.0 and I found that the backup function plays a song (maybe hip pop style) while the backup in going on.
Is that an OmniNote issue or is my phone that has been hacked?

Bye

**Comment by federicoiosue** [comment]
![bitmoji](https://sdk.bitmoji.com/render/panel/4773aa63-7a56-4120-9c33-019fd70dd876-60b8a5dc-65eb-468b-a8ee-e63c10a1dc4c-v1.png?transparent=1&palette=1&width=246)![bitmoji](https://sdk.bitmoji.com/render/panel/4773aa63-7a56-4120-9c33-019fd70dd876-60b8a5dc-65eb-468b-a8ee-e63c10a1dc4c-v1.png?transparent=1&palette=1&width=246)

Surelly not an Omni Notes' feature but actually not that much dangerous as a hack!
It's really strange, would you mind recording a screencast with auto to give a look?

**Comment by bondlxv** [comment]
Ciao Federico,
ti invio la registrazione dello schermo di OmniNotes durante il backup
dell'archivio.
Non so perché parta quella canzone ma se deriva dalla tua app, posso
assicurarti che con la versione precedente non succedeva.

Grazie per la tua bella applicazione e sono contento che tu l'abbia
aggiornata.

Ciao.

Il gio 14 apr 2022, 23:45 Federico Iosue ***@***.***> ha
scritto:

**Comment by federicoiosue** [comment]
I'm afraid you missed attaching the screen recording!

**Comment by bondlxv** [comment]
Sorry for answering in Italian. I thought we started a private conversation via email, by which I sent you an attachment containing a screen and audio recording of the issue.
By the way, I realized that the OmniNotes app I had was a beta version so I did a reinstallation using Google's PlayStore and now the backup function plays only a notification sound.
I think we can close this issue, although a doubt still remains: why hip pop? I prefer rock! 😁
